github/ci/prow-deploy/files/jobs/kubevirt/kubevirtci/kubevirtci-presubmits.yaml

```diff
@@ -81,6 +81,54 @@ presubmits:
           path: /dev/vfio/
           type: Directory
         name: vfio
+  - always_run: false
```

Comment:
do we want it always run ?

Reply:
I do. Not sure what's the criteria though.

Reply:
done, we can do run if changed instead as it depends only on those two files

Reply:
used "run if changed" so only if the folder changed it will run
no need for every unrelated file change

Reply:
https://github.com/kubevirt/project-infra/pull/3430/files#diff-ab108ff0d13755bd671fe2f0a2397f0c0290dc6655d2a03659b2cee3fa5958f7R85

the folder with this new files that were added on the PR

Reply:
so, let me see if I understand ... we'll only run the tests if the files in this folder change. 

I don't see any sense in that. Is that because this is a draft ? 

... should I even be looking at this ?

Reply:
it is not because it is a draft and it is something common practice we are using already.
if the files in the folder aren't changed there is no need to run the lane on every PR

@brianmcarey wdyt?

Reply:
So if the source code that we want to test changes (on KubeVirt) we don't test it. 

If the lane configuration files changes do, we do test it. 

That's how I'm reading this. Hence I'm finding it very strange. I hope I'm wrong.

Reply:
this is kubevirtci lane, not kubevirt, maybe this what confuse you

(see file name please github/ci/prow-deploy/files/jobs/kubevirt/kubevirtci/kubevirtci-presubmits.yaml)

it means that only if those files on kubevirtci repo are changed (which are the ones that affect this provider)
it will rerun the lane
if you change vm based providers (k8s-1.28 etc), it wont run it

everytime a PR is merged on kubevirtci, and bump will occurs on kubevirt, taking the changes
this lane doesn't affect kubevirt testing at all
for kubevirt we will have other lane, which doesn't have this condition but always runs

Reply:
OK I see. Disregard my ignorant comment.